custom_components/localtuya/__init__.py

```diff
@@ -33,6 +33,8 @@
       - platform: light
         friendly_name: Device Light
         id: 4
+        brightness: 20
+        color_temp: 21
```

Comment:
Shouldn't we be using color_temperature in the extended format, as it has been done for set_position in covers?